bazel/google_cloud_cpp_deps.bzl

```diff
@@ -229,3 +229,13 @@ def google_cloud_cpp_deps(name = None):
         strip_prefix = "pugixml-1.13",
         build_file = Label("//bazel:pugixml.BUILD"),
     )
+
+    maybe(
+        http_archive,
+        name = "com_github_jbeder_yaml_cpp",
+        urls = [
+            "https://github.com/jbeder/yaml-cpp/archive/refs/tags/yaml-cpp-0.7.0.tar.gz",
```

Comment:
Do we want to cache it in GCS?

Reply:
Yes.  Funny thing, I had already created the copy and everything.

Reply:
If you add the alternative URL first, you should just be able to `bazel/deps-cache.py -p`.